packages/trace-viewer/src/ui/actionList.tsx

```diff
@@ -111,3 +112,8 @@ export const renderAction = (
     </div>}
   </>;
 };
+
+function excludeBaseUrl(url: string): string {
+  const match = url.match(/^(https?:\/\/[^/]+)(.*)$/);
+  return match ? match[2] : url;
+}
```

Comment:
```suggestion
function excludeOrigin(url: string): string {
  try {
    const urlObject = new URL(url);
    return urlObject.pathname + urlObject.search + urlObject.hash;
  } catch (error) {
    // If the URL is invalid, return the original input
    return url;
  }
}
```

I think we want something like that which is simpler to understand.